modules/gapi/test/gapi_fluid_test.cpp

```diff
@@ -244,6 +244,45 @@ TEST(Fluid, Sum_2_Mats_And_Scalar)
     EXPECT_EQ(0, cv::countNonZero(out_mat != ref_mat));
 }
 
+TEST(Fluid, EqualizeHist)
+{
+    cv::GMat mat;
+    cv::GArray<int> arr;
+
+    cv::GComputation c(cv::GIn(mat, arr), cv::GOut(TEqualizeHist::on(mat, arr)));
```

Comment:
Also it would be a killer feature to test if the histogram itself was calculated in the graph.
You might need to write an extra OCV kernel to turn it into probability function though

Reply:
Where do you suggest to write the kernel? Just somewhere inside tests or add it to imgproc, for example?

Reply:
Done